src/SwedbankPay/Api/Client/ClientVersion.php

```diff
@@ -235,29 +233,46 @@ private function tryReadComposerJson(&$decodedJsonObject) : bool
      */
     private function tryReadComposerLock(&$decodedJsonObject, $path = '') : bool
     {
-        if ($path == '') {
-            $path = $this->getComposerPath() . DIRECTORY_SEPARATOR . 'composer.lock';
+        if (!is_readable($path)) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The use of function is_readable() is discouraged](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=5631375)